### PR TITLE
Copyedit security-keycloak-authorization.adoc

### PR DESCRIPTION
**Pull request by rolfedh**
This PR includes copy edits to enhance clarity, improve consistency, and align the content with style guidelines for a better reader experience.

### CODE REVIEW

File: docs/src/main/asciidoc/security-keycloak-authorization.adoc [hunk 1]
```diff
@@ -1,39 +1,62 @@
...
+3. Allows Keycloak to evaluate resource-based permissions dynamically, by using attributes such as resource name, identifier, or URI.
+
+By externalizing authorization decisions, you can:
+
+- Implement diverse access control strategies without modifying application code.
+- Reduce redeployment needs as security requirements evolve.
+
+.Compatibility
+
+This extension is compatible only with Quarkus xref:security-oidc-bearer-token-authentication.adoc[OIDC service applications]. It replaces explicit mechanisms such as role-based access control with dynamic authorization policies.
```
Comment (michalvavrik) [inline docs/src/main/asciidoc/security-keycloak-authorization.adoc:39]:
```suggestion
This extension is compatible only with Quarkus xref:security-oidc-bearer-token-authentication.adoc[OIDC service applications]. It complements explicit mechanisms such as role-based access control with dynamic authorization policies.
```

Comment (rolfedh) [reply docs/src/main/asciidoc/security-keycloak-authorization.adoc:39]: Hi @michalvavrik,

Thank you for your feedback! I’ve applied the suggested change. If you have any additional thoughts or suggestions, please feel free to share—your input is always greatly appreciated.

Just a heads-up: this PR is still a work in progress, so I’ll be refining the existing changes and adding new ones over the coming week. Stay tuned for updates, and thanks again for your support!

Comment (michalvavrik) [reply docs/src/main/asciidoc/security-keycloak-authorization.adoc]: Hey Rolfe, no, sorry I don't really have a time ATM, I just mentioned your PR and thought that Kc AuthZ shouldn't be saw like replacement for RBAC annotations and HTTP perms etc. I'll try to read your PR when I have a time, but please don't wait for me.

**Review by sberyozkin — APPROVED** [review]
Nicely done, thanks @rolfedh

**Comment by sberyozkin** [comment]
@rolfedh Please enable it for review, so that we can eventually merge

**Comment by rolfedh** [comment]
Thanks, @sberyozkin. I've enabled it for review now.  

This document is quite extensive, and I plan to continue working on the remaining sections. Feel free to merge this version as it stands, and I'll follow up with additional PRs for the subsequent updates.  

Let me know if you have any concerns or specific feedback!

**Comment by sberyozkin** [comment]
Sounds good, thanks @rolfedh, it would be easier to handle it iteratively then, thanks

**Pull request merged**